run_tests_coverage.sh

```diff
@@ -8,7 +8,11 @@ while [ $count -le $numOfModes ]
 do
     echo "Run mode $count"
     # Run tests under tests/ and rules/
-    coverage run --parallel-mode -m pytest tests/ rules/*.py --mode $count
+    coverage run --parallel-mode -m pytest pdsfile/pds3file/tests/ \
```

Comment:
Let's put this on one line

Reply:
Also, why not include pds4file on the same line? Or just say pdsfile by itself, won't it find all of the subdirectory test files then?

Reply:
The reason we separate pds3 & pds4 tests into two coverage commands is because we want to make sure different conftest (setup) files are run before each testing. We want to make sure for pds3, we preload pds3 test directory, and for pds4, we preload pds4 test directory.

If we just put both pds3 & pds4 into one coverage command, there will be an error.

Reply:
Now there is only one coverage pytest command being run.